index.html

```diff
@@ -0,0 +1,104 @@
+
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+
+</head>
+
+<body>
+    <!-- <h1>Hello World, we are software developers</h1> -->
+    <header id="header">
+        <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo" >
+        <nav id="nav-bar">
+            <a class="nav-link" href="#premium">Features</a>
+            <a class="nav-link" href="#fast">How It Works</a>
+            <a class="nav-link" href="#quality">Pricing</a>
+        </nav>
+    </header> 
+    <main class="landingPage">
+        <h1> Handcrafted, home-made masterpieces</h1>
+
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input id="email" name="email" type="email" class="email" placeholder="Enter your email address" required>
+            <input id="submit" type="submit" class="submit" >
+    </form> 
+</main>
+
+    <section >
+    <article id="premium">
+        <i class="fa fa-3x fa-fire" style="color:orange;"></i>
+        <div class="article-bar">
+            <h1>Premium Materials</h1>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        </div>
+    </article>
+
+    <article id="fast">
+        <i class="fa fa-3x fa-truck" style="color:orange;"></i>
+        <div  class="article-bar"> 
+            <h1>Fast Shipping</h1>
+            <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        </div>
+    </article>
+
+    <article id="quality">
+        <i class="fa fa-3x fa-battery-full" style="color:orange;"aria-hidden="true"></i>
+        <div  class="article-bar">
+            <h1><a href="#pricing"></a>Quality Assurance</a></h1>
+            <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+        </div>
+    </article>
+
+</section>
+
+<iframe id="video" width="600" height="340" src="https://www.youtube.com/embed/W6NZfCO5SIk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="articles">
+    <section class="side-band">
+        <h3>TENOR TROMBONE</h3> 
+        <strong>$600</strong>
+        <p>Lorem ipsum.Lorem ipsum.Lorem ipsum dolor.Lorem ipsum.</p>
+        <button>SELECT</button>
+    </section>
+
+    <section class="side-band">
+        <h3>BASS TROMBONE</h3>
+        <strong>$900</strong>
+        <p>Lorem ipsum.Lorem ipsum.Lorem ipsum dolor.Lorem ipsum.</p>
+        <button>SELECT</button>
+    </section>
+
+    <section class="side-band">
+        <h3>VALVE TROMBONE</h3>
+        <strong>$1200</strong>
+        <p>Lorem ipsum.Lorem ipsum.Lorem ipsum dolor.Lorem ipsum.</p>
+        <button>SELECT</button>
+    </section>
```

Comment:
- [OPTIONAL] For this section you could use a `ul` tag and your inner sections become `li` of the `ul` , it's always a good pratics to use `li` to represent an item in a list and use a ` ul` tag to group them